services/madoc-ts/src/gateway/tasks/import-manifest.ts

```diff
@@ -90,6 +90,25 @@ export const jobHandler = async (name: string, taskId: string, api: ApiClient) =
       // 2. Save manifest to disk, IF it does not already exist.
       const fileLocation = await tasks.saveManifestToDisk(idHash, text);
 
+      // trim very long strings to 3000 characters
+      const label = iiifManifest.label;
+      const summary = iiifManifest.label;
+      const metadata = iiifManifest.metadata;
+
+      const trimIntString = (intString: InternationalString) => {
+        for (const [key, value] of Object.entries(intString)) {
+          if (value) {
+            value[0] = value[0].substring(0, 3000);
+            return intString;
+          }
+        }
+        return intString;
+      };
```

Comment:
We might want to do this for collections/canvases later, might be worth splitting out to the helper folder